.gitignore

```diff
@@ -28,6 +28,7 @@
 /test/case
 /test/iscase
 /test/custom
+/test/app/build
```

Comment:
I'm not sure we need this.

@eschnett Do you want to add this? If we want to use `cmake -S test/app -B test/app/build ...` for local development, this will be helpful. If we want to use `cmake -S test/app -B ../test-app-build ...` or something for local development, this is needless.

FYI: Our README has `cmake -S . -B build` but we don't have `/build/` in this list.
https://github.com/JuliaStrings/utf8proc?tab=readme-ov-file#quick-start

Reply:
I would not add this. I would only add directories or files that are generated by the build system. The path where cmake builds is under user control and I frequently use different paths than those suggested in a packages's documentation. I have, however, no strong opinion on this.